contributing/ci-bio-formats.txt

```diff
@@ -140,12 +140,11 @@ The branch for the 5.1.x series of Bio-Formats is develop.
 
 		This job builds the merge generated sources of Bio-Formats
 
-		#. Checks out the develop/merge/daily branch of the
-		   snoopycrimecop fork of bioformats.git_
+		#. Checks out :bf_scc_branch:`develop/merge/daily`
 		#. Builds the Java generated sources using ``ant clean ome-xml-src``
 		#. Builds the C++ generated sources using ``cmake .. && make gensrc``
-		#. Commits all the generated sources and pushes to the
-		   develop/merge/generated-sources branch
+		#. Commits all the generated sources and pushes to
+		   :bf_scc_branch:`develop/merge/generated_sources` branch
 		#. Computes the diff with develop/latest/generated-sources and archives
```

Comment:
Very minor nitpick, but shouldn't this be 'develop/latest/generated_sources with an underscore given it is referring to the branch? And do you want to make this one a link too?